test/test_other.py

```diff
@@ -2972,8 +2972,8 @@ def test_embind_closure_no_dynamic_execution(self):
   })
   @parameterized({
     '': [],
-    # Ensure embind compiles under C++17 where "noexcept" became part of the function signature.
-    'cxx17': ['-std=c++17'],
+    # Ensure embind compiles under C++11 which is the miniumum supported version.
+    'cxx11': ['-std=c++11'],
```

Comment:
Should we run this in c++17 and c++20 too?

Reply:
17 is currently the default. 20 would be good but it currently fails. I've filed https://github.com/llvm/llvm-project/issues/78614 to look into it.

Reply:
Perhaps mention this in comments?  i.e. `# With no arguments we are effectively testing c++17 since it is the default`